qml/pages/About.qml

```diff
@@ -75,7 +75,7 @@ Page {
             }
 
             Label {
-                //% "ApostolosB, dashinfantry, flypig, smdesai, spodermenpls, Thaodan"
+                //% "ApostolosB, bionade24, dashinfantry, flypig, smdesai, spodermenpls, Thaodan"
```

Comment:
@bionade24: I've added your nick, but would you prefer your full name here?

Reply:
It's fine. Would be weird if I'm the only one with real name there.

Reply:
It's completely your call, and I think it would look fine with a full name myself. Some people don't want their real name used, but I'm sure if you use your full name, others will join you doing it the same way in the future.

Reply:
Ok then put `Oskar Roesler (bionade24)` there.